### DISCUSSION TITLE
Implementing Bi-directional Custom Datalink over Telemetry Stream without IPoWB

### DISCUSSION THREAD

**Discussion by akhaldi** [svpcom/wfb-ng#302] [Questions]
I'm looking for a method to set up bi-directional custom datalink via a telemetry stream. Messages should originate from a PC and terminate at a companion computer, passing through a ground RPi and a drone RPi along the way.

The intent is to create a bi-directional communication line that bypasses the IPoWB (IPv4 over Wifibroadcast tunnel), as I've found it to be slow and unstable. 

Could you suggest any reliable alternatives for this specific use case? Any guidance would be greatly appreciated.

**Notes**: 
- If I were to use a tunnel, I would typically employ gRPC for sending/receiving data.
- The message size is very small; it's just for changing settings and issuing some commands.

Thank you in advance

**Comment by relaxibus** [comment]
I have the same question using IP based radios on non RPi boards. IP based radios could be like Microhard or Doodle Labs. My platform is a NXP iMX8M Plus on the air and a x86 router on the ground.
